handler.py

```diff
@@ -11,6 +11,7 @@
 
 import numpy as np
 from raster_processing import *
+from to_agol import *
```

Comment:
Star imports are terrible. I've been meaning to get rid of the existing ones, but let's not include new ones.

Reply:
Completed with commit c80429f